### PR TITLE
Added my Balance 12 layout for Atreus

### PR DESCRIPTION
**Pull request by yttyx**
The first Balance 12 layout in the QMK repository?

### CODE REVIEW

File: keyboards/atreus/keymaps/yttyx/config.h [hunk 1]
```diff
@@ -0,0 +1,59 @@
+#ifndef CONFIG_H
```
Comment (drashna) [inline keyboards/atreus/keymaps/yttyx/config.h]: You shouldn't need all of this.  Only the stuff that you've customized.

File: keyboards/atreus/keymaps/yttyx/keymap.c [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  */
+  [R3] = LAYOUT(
+    RESET,   xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx,                   xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx, _______,
+    xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx,                   xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx,
+    xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx,                   xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx, 
+    xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx, xxxxxxx
+  )
+};
+
+const uint16_t PROGMEM fn_actions[] = {};
```
Comment (drashna) [inline keyboards/atreus/keymaps/yttyx/keymap.c]: If you've disabled these features, you should remove these functions

File: keyboards/atreus/keymaps/yttyx/README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+* The central column of punctuation keys has been moved elsewhere
+* Home positions for the left and right forefingers are *T* and *A* respectively
+
+## To build
+
+> make atreus:yttyx
+
+## To flash
+
+> avrdude -p atmega32u4 -c avr109 -U flash:w:atreus_yttyx.hex -P /dev/ttyACM0
```
Comment (drashna) [inline keyboards/atreus/keymaps/yttyx/README.md]: `make atreus:yttyx:avrdude` should take care of this automatically (include port detection, IIRC)

**Comment by yttyx** [comment]
Thank you! - I have made the following changes:
- Removed unnecessary config.h lines
- Removed unused stub functions from keymap.c
- Modified layout build instructions

**Comment by drashna** [comment]
Awesome, thanks!  Looks good!

**Comment by yttyx** [comment]
Much appreciated - thank you!

**Pull request merged**